### ISSUE TITLE
Simplify npm script test

### ISSUE THREAD

**Issue by megoth** [solid/solid-ui#287]
Today we run `npm run build` as part of `npm test` so that Travis is able to run the code properly. I propose we simplify the build-part to only make sure that versionInfo is built, and then modify Travis to do the extra steps necessary to allow it to what it needs to do.

**Comment by michielbdejong** [comment]
As discussed in Slack, I think the second option (modify Travis to do the extra steps necessary to allow it to what it needs to do) and then removing `npm run build` from `npm test` altogether, is the best approach.

**Comment by bourgeoa** [comment]
Does it imply that you cannot test locally ?

**Comment by michielbdejong** [comment]
Not really, we can specify in the README that the way to run the tests is:
```sh
npm install
npm run build
npm test
```

**Comment by michielbdejong** [comment]
But yeah, just running `npm i && npm test` would no longer work.

**Comment by megoth** [comment]
The only thing we need to build in order to make the tests run are `versionInfo`, so please just change the build-step in test into `build-versionInfo` or something, and then it should be good.
